easybuild/easyconfigs/p/Python/Python-3.6.4-golf-2018a.eb

```diff
@@ -0,0 +1,209 @@
+name = 'Python'
+version = '3.6.4'
+
+homepage = 'http://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'golf', 'version': '2018a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['7dc453e1a93c083388eb1a23a256862407f8234a96dc4fae0fc7682020227486']
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.11'),
+    ('libreadline', '7.0'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.21.0'),
+    ('XZ', '5.2.3'),
+    ('GMP', '6.1.2'),  # required for pycrypto
+    ('libffi', '3.2.1'),  # required for cryptography
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.0g'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
+
+# workaround for "undefined symbol: __stack_chk_guard"
+# see also https://software.intel.com/en-us/forums/intel-c-compiler/topic/610514
+buildopts = 'LDFLAGS="$LDFLAGS -lssp"'
```

Comment:
@bartoldeman This is not required since we're not building with Intel compilers?